sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/csv/CSVSuite.scala

```diff
@@ -1941,7 +1963,10 @@ abstract class CSVSuite
             .csv(path.getCanonicalPath)
             .collect()
         }
-        assert(exception.getErrorClass.startsWith("FAILED_READ_FILE"))
```

Comment:
Obviously, this code should test `exceptionForShortSchema` instead of `exception`